src/pages/components/PageDetailsPage/PageDetailsPage.tsx

```diff
@@ -142,148 +143,136 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
-                disabled={loading}
-                messages={{
-                  hiddenLabel: intl.formatMessage({
-                    id: "/TK7QD",
-                    defaultMessage: "Hidden",
-                    description: "page label",
-                  }),
-                  hiddenSecondLabel: intl.formatMessage(
-                    {
-                      id: "GZgjK7",
-                      defaultMessage: "will be visible from {date}",
-                      description: "page",
-                    },
-                    {
-                      date: localizeDate(data.publicationDate),
-                    },
-                  ),
-                  visibleLabel: intl.formatMessage({
-                    id: "X26jCC",
-                    defaultMessage: "Visible",
-                    description: "page label",
-                  }),
-                }}
-                onChange={change}
-              />
-              <CardSpacer />
-              <PageOrganizeContent
-                data={data}
-                errors={errors}
-                disabled={loading}
-                pageTypes={pageTypes}
-                pageType={data.pageType}
-                pageTypeInputDisplayValue={data.pageType?.name || ""}
-                onPageTypeChange={handlers.selectPageType}
-                fetchPageTypes={fetchPageTypes}
-                fetchMorePageTypes={fetchMorePageTypes}
-                canChangeType={!page?.pageType}
-              />
-            </div>
-          </Grid>
-          <Savebar
-            disabled={isSaveDisabled}
-            state={saveButtonBarState}
-            onCancel={() => navigate(pageListUrl())}
-            onDelete={page === null ? undefined : onRemove}
-            onSubmit={submit}
-          />
-          {canOpenAssignReferencesAttributeDialog && (
-            <AssignAttributeValueDialog
-              entityType={getReferenceAttributeEntityTypeFromAttribute(
-                assignReferencesAttributeId,
-                data.attributes,
-              )}
-              confirmButtonState={"default"}
-              products={referenceProducts}
-              pages={referencePages}
-              hasMore={handlers.fetchMoreReferences?.hasMore}
-              open={canOpenAssignReferencesAttributeDialog}
-              onFetch={handlers.fetchReferences}
-              onFetchMore={handlers.fetchMoreReferences?.onFetchMore}
-              loading={handlers.fetchMoreReferences?.loading}
-              onClose={onCloseDialog}
-              onSubmit={attributeValues =>
-                handleAssignReferenceAttribute(attributeValues, data, handlers)
-              }
+                <CardSpacer />
+                <SeoForm
+                  errors={errors}
+                  allowEmptySlug={!pageExists}
+                  description={data.seoDescription}
+                  disabled={loading}
+                  descriptionPlaceholder={""} // TODO: Cast description to string and trim it
```

Comment:
?? just noticed that - can we create a ticket for it? (maintenance / fast lane)

Reply:
Yep, https://app.clickup.com/t/2wguf9b